test/testcafe/framework/page-objects/SuccessPageObject.js

```diff
@@ -6,7 +6,7 @@ export default class SuccessPageObject extends BasePageObject {
     super(t);
   }
   async getPageUrl() {
-    await this.t.expect(Selector('h1').innerText).eql('Mock User Dashboard');
+    await this.t.expect(Selector('#mock-user-dashboard-title').innerText).eql('Mock User Dashboard');
```

Comment:
After adding the `h1` to the Header, testcafe was picking up on that when calling this selector rather than the later `h1` that is on the mock user home landing page of the redirect (`app/userHome` above)